src/ontology/mondo-edit.obo

```diff
@@ -92598,8 +92598,11 @@ xref: MESH:D004681 {source="EFO:0001066", source="MONDO:equivalentTo"}
 xref: NCIT:C3006 {source="EFO:0001066", source="MONDO:otherHierarchy"}
 xref: SCTID:27040004 {source="EFO:0001066"}
 xref: SCTID:50776006 {source="EFO:0001066"}
-is_a: MONDO:0005156 {source="EFO:0001066"} ! encephalomyelitis
-is_a: MONDO:0006704 {source="MESH:D004681"} ! CNS demyelinating autoimmune disease
+is_a: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+is_a: MONDO:1011336 {source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+relationship: excluded_subClassOf MONDO:0005156 {source="EFO:0001066", source="https://orcid.org/0000-0002-5002-8648"} ! encephalomyelitis
+relationship: excluded_subClassOf MONDO:0006704 {source="MESH:D004681", source="https://orcid.org/0000-0002-5002-8648"} ! CNS demyelinating autoimmune disease
```

Comment:
@katiermullen adding your orcid for this "excluded_subclassOf" means that you were a source for the subclass (ie, you originally said that this term should be a subclass of encephanlomyelitis/CNS demyelinating autoimmune disease). 
Therefore you should not add this source. 
I can update for you, but first, @nicolevasilevsky please confirm that I am not making things up.

Reply:
Whichever is correct, can we also add this to the documentation: https://mondo.readthedocs.io/en/latest/editors-guide/revise-terms-in-mondo/#exclude-a-superclass?

Reply:
I think it's meant to mean that you reviewed this annotation and agree with it, ie you agree that CNS demyelinating autoimmune disease should not be a superclass of this class

Reply:
@twhetzel yes! let's add it to the documentation

Reply:
I think it could really be interpreted either way though.

This came about from this ticket: https://github.com/monarch-initiative/mondo/issues/7114

Reply:
From technical call on 2024-03-29: 
yes, this is how it should be done, ie using "source" for both the "original source" and "orcid of the curator who OKed the SC exclusion". It might change in the future, but it is how we do right now. 

I will accept and merge this PR.